t/unit/tasks/test_canvas.py

```diff
@@ -1019,15 +585,11 @@ def s(*args, **kwargs):
         assert c.tasks[-1].options['chord'] == 'some_chord_id'
 
         c.apply_async(link=[s(32)])
-        expected_sig = s(32)
-        expected_sig.stamp(visitor=GroupStampingVisitor())
```

Comment:
This was an API breaking change the `GroupStampingVisitor` introduced that could have broken prod systems post 5.3 release...